segment_point_cloud_topic: "/depth_segmentation_node/object_segment"
world_frame_id: "world"
camera_frame_id: "depth"
scene_gsm_update_topic: "all_gsm_updates"
segment_gsm_update_topic: "all_gsm_updates"
object_flushing_age_threshold: 6
validate_merged_object: "validate_merged_object"
enable_pairwise_confidence_merging: false
pairwise_confidence_ratio_threshold: 0.2
pairwise_confidence_count_threshold: 30
mesh_filename: "gsm_tango_mesh.ply"
publish_segment_mesh: false
publish_scene_mesh: false
voxel_size: 0.01
voxels_per_side: 8
truncation_distance_factor: 5.0
voxel_carving_enabled: false
max_ray_length_m: 2.5

sliding_window:
  is_on: true
  removed_segment_label_topic: removed_segment_label
  radius: 1.0
  update_period: 10.0

save_scene_mesh: false
publish_all_segments: true
